nodepool/config/config.go

```diff
@@ -148,6 +150,22 @@ func ClusterFromBytes(data []byte) (*ProvidedConfig, error) {
 		c.InstanceCIDR = "10.0.1.0/24"
 	}
 
+	//Computed defaults
+	launchSpecs := []LaunchSpecification{}
+	for _, spec := range c.Worker.SpotFleet.LaunchSpecifications {
+		if spec.RootVolumeType == "" {
+			spec.RootVolumeType = c.WorkerRootVolumeType
+		}
+		if spec.RootVolumeSize == 0 {
+			spec.RootVolumeSize = c.WorkerRootVolumeSize * spec.WeightedCapacity
```

Comment:
I'd rather like to make this computed via `c.WorkerSpotFleet.UnitRootVolumeSize * spec.WeightedCapacity` while adding `UnitRootVolumeSize` to the SpotFleet struct, so that we can be more explicit that it is not the exact size of EBS volumes but size per "unit".